tests/unit/dom/MediaElement.test.js

```diff
@@ -0,0 +1,43 @@
+/* global assert, beforeEach, describe */
+const fs = require('fs');
+const path = require('path');
+const exokit = require('../../../index');
+
+const audioData = fs.readFileSync(path.resolve(__dirname, '../data/test.ogg'), 'base64');
+const audioUri = `data:audio/ogg;base64,${audioData}`;
+
+const imageData = fs.readFileSync(path.resolve(__dirname, '../data/test.png'), 'base64');
+const imageUri = `data:image/img;base64,${imageData}`;
+
+const videoData = fs.readFileSync(path.resolve(__dirname, '../data/test.mp4'), 'base64');
+const videoUri = `data:video/mp4;base64,${videoData}`;
+
+describe('MediaElement', () => {
+  var document;
+  var el;
+
+  beforeEach(() => {
+    document = exokit().document;
+  });
+
+  describe('<video>', () => {
+    it('can set src', () => {
+      el = document.createElement('video');
+      el.setAttribute('src', videoUri);
```

Comment:
Should probably wait for `onload`/`onerror`. Otherwise a whole class of bugs potentially fires after the test has passed.

Reply:
Sorry, for video/audio I think it's `oncanplay`.